pages/design/test-cases.md

```diff
@@ -0,0 +1,70 @@
+# ACT Test Case Design
+
+The goal of test cases in ACT rules is to allow implementers to verify that their implementation is consistent with the ACT rule. This requires that a rule has test cases for all of the "important" aspects of it. Beyond that, it is up to the rule authors to determine how extensive to make the list of test cases. A list of test cases full of edge cases would ensure greater consistency between implementers, but may result in fewer implementers deciding to adopt the rule.
```

Comment:
This description seems more discouraging than what we have discussed. The entire purpose of ACT is to ensure greater consistency between implementations, so challenging that seems counter-productive. Can we say this more positively, that we encourage the broadest coverage possible but that this needs to be based on real-world issues (as opposed to theoretical situations that do not occur in practice)?